src/internet/model/tcp-bbr-v2.cc

```diff
@@ -560,10 +561,22 @@ TcpBbrV2::ModulateCwndForProbeRTT(Ptr<TcpSocketState> tcb)
     NS_LOG_FUNCTION(this << tcb);
     if (m_state == BbrMode_t::BBR_PROBE_RTT)
     {
-        tcb->m_cWnd = std::min(tcb->m_cWnd.Get(), m_minPipeCwnd);
+        tcb->m_cWnd = std::min(tcb->m_cWnd.Get(), ProbeRTTCwnd(tcb));
     }
 }
 
+uint32_t
+TcpBbrV2::ProbeRTTCwnd(Ptr<TcpSocketState> tcb)
+{
+    NS_LOG_FUNCTION(this << tcb);
+
+    if (m_probeRttCwndGain == 0)
+    {
+        return m_minPipeCwnd;
+    }
+    return std::max(m_minPipeCwnd, GetBDP(tcb, m_probeRttCwndGain));
+}
```

Comment:
Field `m_probeRttCwndGain` = 1/2. It is multiplier for BDP/2, cwnd size in ProbeRTT